superbench/config/azure_ndv4.yaml

```diff
@@ -35,6 +35,25 @@ superbench:
       <<: *default_local_mode
     gemm-flops:
       <<: *default_local_mode
+    disk-benchmark:
+      enable: false
+      modes:
+        - name: local
+          proc_num: 1
+          parallel: no
+      parameters:
+        block_devices: []
+    gpu-sm-copy-bw:
+      enable: false
```

Comment:
For `gpu-sm-copy-bw`, I think we can enable it by default if it works fine.

Reply:
It takes around 5min to finish on a 4 NUMA 8 GPU machine. We can enable it by default if it's acceptable. What do you think?

Reply:
I think Guoshuai mean that you should enable it. Otherwise, we should remove it.

Reply:
Fixed.